modules/Time.js

```diff
@@ -0,0 +1,7 @@
+import { DateTime } from '../node_modules/luxon/build/es6/luxon.js';
+
+export default () => {
+  const now = DateTime.now();
+  const liveTime = document.getElementById('live-time');
+  liveTime.innerHTML = ` ${now}   `;
+};
```

Comment:
- [ ] [OPTIONAL] Good job with this feature but I think is not entirely humanly friendly now in terms of readability, design, and user experience, looks more like some cryptic ASCII CODE meant for machines to read :smile: 
- _See it here:_
![image](https://user-images.githubusercontent.com/99715640/235532719-efd79b52-5b38-496e-8627-ebacc2f62e19.png)

  - If I may suggest, I think the major information that users need to see here is just the date (yyyy/mm/dd) and standard time (hh/mm/ss). These would do, I think :thinking: 
  - Another thing that might help increase your user experience is to place the "time-telling machine" - piece of code in a callback  function. E.g. `setInterval(function, 1000)` so that your time can stay updated without a page reload or refresh. :thought_balloon: What do you think? 

_**P.S: Don't forget that this is an optional suggestion to make your code even better :wink:**_